saltedkey/saltedkey_test.go

```diff
@@ -0,0 +1,79 @@
+package saltedkey
+
+import (
+	"crypto/rand"
+	"fmt"
+	"io"
+	"math/big"
+	"testing"
+
+	"github.com/arnaucube/go-blindsecp256k1"
+	blind "github.com/arnaucube/go-blindsecp256k1"
+	qt "github.com/frankban/quicktest"
+	"go.vocdoni.io/dvote/crypto/ethereum"
+)
+
+func TestECDSAsaltedKey(t *testing.T) {
+
+	privHex := fmt.Sprintf("%x", randomBytes(32))
+	sk, err := NewSaltedKey(privHex)
+	qt.Assert(t, err, qt.IsNil)
+
+	salt := [SaltSize]byte{}
+	copy(salt[:], randomBytes(20))
+	msg := []byte("hello world!")
+
+	signature, err := sk.SignECDSA(salt, msg)
+	qt.Assert(t, err, qt.IsNil)
+
+	signingKeys := ethereum.NewSignKeys()
+	err = signingKeys.AddHexKey(privHex)
+	qt.Assert(t, err, qt.IsNil)
+
+	pubKeySalted, err := SaltECDSAPubKey(signingKeys.PublicKey(), salt)
```

Comment:
the `err` is not being checked. I've tried locally adding the `err` check, and it is returning the error `invalid secp256k1 public key`.